### DISCUSSION TITLE
"WEBUI_AUTH = False" environment without docker

### DISCUSSION THREAD

**Discussion by melody-cheung520** [open-webui/open-webui#4596] [Ideas]
In case you wondering, where do I add the "WEB_UI = False" argument without docker. Here's the instructions:
1. Go to ~/open-webui/backend
2. Nano or Xcode open config.py
3. Find WEBUI_AUTH = os.environ.get("WEBUI_AUTH", "True").lower() == "true" or navigate to line 307
4. Change  "true" to "false"
5. Save it and re-run open-webui

And you done! When you log inside the interface, you can start your chatbox without logging in. Have fun.
(PS: This tutorial mainly for macOS, but I think its the same place on Windows)
(Reference: [Running docker container with env flag "ENABLE_SIGNUP="False"" still shows Signup form #1876](https://github.com/open-webui/open-webui/discussions/1876))
![image](https://github.com/user-attachments/assets/27c089e0-b944-443a-b236-3e0263512685)

**Comment by CurtisDS** [comment]
Im on windows. I set the environment variable... but the web ui still asks me to create a log in on my fresh install?

**Reply by melody-cheung520** [reply]
I think environment variable has no relation with the open-webui directory that you installed everytime. For me, I just edit it inside /backend/config.py and it skips the login prompt.

**Comment by NicholasDewberryOfficial** [comment]
Thanks for posting this, but I still have an issue. I'm trying to run this locally using UV, but it hangs on the main page saying that my login failed.

I didn't see anything in the config.py around the WEBUI_AUTH variable, so I manually added this to the webui section: 

`####################################
# WEBUI_AUTH (Required for security)
####################################


WEBUI_AUTH = os.environ.get("WEBUI_AUTH", "True").lower() == "false"
`
 

![image](https://github.com/user-attachments/assets/eabf8906-2517-40f8-9ac6-9855bbb12a4d)

Any clues how to fix this? I can provide the config.py if necessary. 

Running on Arch and using UV. My main command is: open-webui serve 

It also fails when I use a static dir or different port (like DATA_DIR=~/.open-webui open-webui serve --port 3000 ) 

Also, this still errors even on a fresh install
